web-api/src/lambdaWrapper.test.ts

```diff
@@ -30,7 +30,9 @@ describe('lambdaWrapper', () => {
         send: jest.fn(),
       }),
     };
-    JSON.parse = jest.fn();
+
+    const orignalParse = JSON.parse;
```

Comment:
If we care about resetting JSON.parse, we need `JSON.parse = originalParse` in an afterAll